src/te/schedule/message_passing.cc

```diff
@@ -62,6 +62,13 @@ void PassDownDomain(const Stage& stage,
     return actx->Simplify(indexdiv(a + (b - 1), b));
   };
 
+  auto minimum = [actx](PrimExpr a, PrimExpr b) {
```

Comment:
The name `minimum` is not exact. This function will only return the minimum Expr when the two Expr can be compared. Otherwise, it will return the latter one. Maybe we can change to `min_or_later` or any other name whose meaning is exact.

Reply:
You are right.  I will make the change.